parser/parser_test.go

```diff
@@ -0,0 +1,59 @@
+package parser_test
+
+import (
+	"github.com/otrego/clamshell/parser"
+	"testing"
+)
+
+func TestCreate(t *testing.T) {
```

Comment:
I'd recommend more extensive tests here especially smaller examples that test specific features. TableTests are a nice way to set this up: https://github.com/golang/go/wiki/TableDrivenTests